src/stylesheets/dsdl/tokens.css

```diff
@@ -49,7 +50,30 @@
   --dsdl-{{ color.group }}-{{ color.level }}: {{ color.value }};
   {% endfor %}
 
-  --calitp-brand-blue: var(--dsdl-blue-70);  /* #045b86 */
-  --calitp-brand-cyan: var(--dsdl-cyan-40);  /* #38abd0 */
+  --calitp-brand-blue:   var(--dsdl-blue-70);    /* #045b86 */
+  --calitp-brand-cyan:   var(--dsdl-cyan-40);    /* #38abd0 */
   --calitp-brand-yellow: var(--dsdl-yellow-30);  /* #febd28 */
 }
+
+
+/* SPACING */
+
+:root {
+  --spacing-1px: 1px;  /* Non-relative unit for when we don't want a single pixel stroke to scale */
```

Comment:
not sure i see the benefit of canonizing this as a CSS variable, but i'm okay with it.

Reply:
I did it to stay in sync with how Christine laid out the spec, but I think you're right that we should ditch it. I agree with the general principle (hence not adding variables for the different font weights before).